internal/commom/extra_files.go

```diff
@@ -0,0 +1,36 @@
+package commom
```

Comment:
I think we can rename the pkg to `extrafiles` and the main function to `Find` only... 

recommented reading: https://blog.golang.org/package-names